lnwallet/channel.go

```diff
@@ -1671,8 +1672,7 @@ func (lc *LightningChannel) restoreStateLogs(
 		// entry within the updateLog, so we'll just apply it and move
 		// on.
 		if feeUpdate, ok := logUpdate.UpdateMsg.(*lnwire.UpdateFee); ok {
-			newFeeRate := btcutil.Amount(feeUpdate.FeePerKw)
-			lc.pendingAckFeeUpdate = &newFeeRate
+			lc.pendingAckFeeUpdate = &feeUpdate.FeePerKw
```

Comment:
was this supposed to be changed?

Reply:
ahh good catch, i think i accidentally edited an older file kept in vim's memory and then saved 😳